bootstrap.sh

```diff
@@ -401,7 +402,8 @@ init_docker() {
     sudo_askpass /bin/chmod 644 /Library/LaunchDaemons/com.docker.vmnetd.plist
     sudo_askpass /bin/launchctl load /Library/LaunchDaemons/com.docker.vmnetd.plist
 
-    open -g -a Docker.app
+    log "About to open Docker.app"
+    open -g -a Docker.app || echo "We were unable to open Docker.app; Try again" && exit 1
```

Comment:
At the core of this PR, all I'm doing is adding the `||`.
I have not been able to find why this step sometimes fails.

Should we just make it a step that can sometimes fail and keep going?

Reply:
I think it's better to fail and have them re-run instead of continuing